jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -152,8 +152,13 @@ function declare_timer_common_variables {
     for var in ${common_vars[@]}; do
         local value=$(find_env "${prefix}_${var}")
         if [[ -n ${value} ]]; then
+          if [ "${var}" = "PASSWORD" ] || [ "${var}" = "USERNAME" ]; then
```

Comment:
any reason to also check the username?

Reply:
To avoid problem if a customer use a username like H@n$0lo, the expansion problem is on credentials used in the files, both username and password are affected, for this reason are escaped EJB_TIMER_USERNAME and EJB_TIMER_PASSWORD